client/src/App.js

```diff
@@ -22,8 +22,12 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
```

Comment:
you should refactor the addToSavedList function....the way it's written here is no longer considered best practice. :)